AppCenter/AppCenterTests/MSServiceNotificationDelegateTests.m

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "MSAppCenter.h"
+#import "MSAppCenterInternal.h"
+#import "MSServiceNotificationDelegate.h"
+#import "MSTestFrameworks.h"
+
+@interface MSServiceNotificationDelegateTests : XCTestCase
+
+@end
+
+@implementation MSServiceNotificationDelegateTests
+
+- (void)testNoRegisteredDelegate {
+  id appCenterMock = OCMPartialMock([MSAppCenter sharedInstance]);
```

Comment:
You'll need to set/reset the AppCenter instance inside the `setup` `teardown` method to run each tests with a clean instance.

Reply:
Note you can also add these tests to the MSAppCenterTests.m file as everything is setup there.

Reply:
I am using MSAppCenterTests.m now for the new tests that I am adding